### PR TITLE
npm install -g errors

### PR DESCRIPTION
**Pull request by matanwrites**
Add link to fix npm install -g EACCESS permissions errors

### CODE REVIEW

File: Solana_NFTs/en/Section_2/Lesson_1_Get_Solana_Running.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ If any of these commands are not found, please make sure to install it before mo
 - [ts-node Installation](https://www.npmjs.com/package/ts-node#installation)
 
 Be sure to install `ts-node` globally. I used this command: `npm install -g ts-node`
+> If you run into EACCES permissions errors while installing, please checkout this [link](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally#manually-change-npms-default-directory)
```
Comment (adilanchian) [inline Solana_NFTs/en/Section_2/Lesson_1_Get_Solana_Running.md:55]: What are your thoughts on changing this link to be: https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally instead?

**Review by adilanchian — CHANGES_REQUESTED** [review]
Hey @matanwrites !

Thanks for the contribution! Left a small comment for ya :). Thanks!

File: Solana_NFTs/en/Section_2/Lesson_1_Get_Solana_Running.md (hunk 1 above)
Comment (matanwrites) [reply Solana_NFTs/en/Section_2/Lesson_1_Get_Solana_Running.md]: Sure, agree let's let users choose however they want to fix it!

**Comment by matanwrites** [comment]
Done! :)

**Review by adilanchian — APPROVED** [review]
Beautiful. Thank you for this!

**Pull request merged**